packages/node/test/watch-service.nodespec.ts

```diff
@@ -97,6 +97,12 @@ describe("Node Watch Service", function () {
       await validator.validateEvents([{ path: testDirectoryPath, stats: null }]);
       await validator.noMoreEvents();
     });
+
+    it("unwatchAllPaths can be invoked immediately after watched dir removal", async () => {
+      await watchService.watchPath(testDirectoryPath);
+      await rmdir(testDirectoryPath);
+      await watchService.unwatchAllPaths();
```

Comment:
Did you verify that this test fails without the fix?

Reply:
The answer is yes.

Reply:
yes